sdks/nodejs/src/agonesSDK.js

```diff
@@ -25,22 +25,8 @@ class AgonesSDK {
 	}
 
 	get port() {
-		const defaultPort = '59357';
 		const port = process.env.AGONES_SDK_GRPC_PORT;
```

Comment:
In JavaScript it's common to use `||` e.g. do `return process.env.AGONES_SDK_GRPC_PORT || '59357'`
Without examining grpc code too much it seems port should be a number so you could cast using `return Number(process.env.AGONES_SDK_GRPC_PORT) || 59357`

Reply:
doh too slow :)

Reply:
I'm happy to fix it in a follow-up PR. :)

Reply:
https://github.com/googleforgames/agones/pull/1129

> it seems port should be a number

The value is returned and concatenated with 'localhost' so I think returning a string is ok.